tools/java/java-build/src/com/google/i18n/phonenumbers/BuildMetadataProtoFromXml.java

```diff
@@ -195,6 +205,16 @@ public boolean start() {
 
       writeCountryCallingCodeMappingToJavaFile(
           countryCodeToRegionCodeMap, outputDir, mappingClass, copyright);
+      
+      if (buildRegioncode) {
+        SortedSet<String> regionCodeSet = new TreeSet<String>();
+        regionCodeSet.addAll(getDefaultRegionCodeList());
+        regionCodeSet.addAll(BuildMetadataFromXml.buildRegionCodeList(metadataCollection));
+        List<String> regionCodeList = new ArrayList<String>(regionCodeSet);
```

Comment:
There is no need to wrap this inside a list, you can just pass a set to the method. You are using a for loop to iterate over it, and since it is a sorted set, the iteration will happen in exactly the same way.